src/CommonExcludedAssets.props

```diff
@@ -21,4 +21,29 @@
     <!-- Microsoft.Bcl.AsyncInterfaces is required by StackExchange.Redis. ExcludeAssets="all" prevents copying it to the output -->
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" ExcludeAssets="all" />
   </ItemGroup>
+
+  <ItemGroup>
+    <!-- DnsClient is required by MongoDB.Driver.Core.Extensions.DiagnosticSources. ExcludeAssets="all" prevents copying it to the output -->
```

Comment:
To avoid the verbosity and the copy/paste mistake it seems better to have a single comment before the `ItemGroup`. Something like "Transient dependencies from MongoDB.Driver.Core.Extensions.DiagnosticSources that are shipped with the applications. ExcludeAssets="all" prevents copying them to the output"

Reply:
Extracted common comment. b7ca218073c744f0b8d980346c47e89224c65dcb
I would keep explanation per library - I expect more to come. Part of them probably can be linked by more than one library.